llama_stack/providers/tests/ci_test_config.yaml

```diff
@@ -0,0 +1,24 @@
+tests:
+- path: inference/test_vision_inference.py
+  functions:
+  - test_vision_chat_completion_streaming
+  - test_vision_chat_completion_non_streaming
+
+- path: inference/test_text_inference.py
+  functions:
+  - test_structured_output
+  - test_chat_completion_streaming
+  - test_chat_completion_non_streaming
+  - test_chat_completion_with_tool_calling
+  - test_chat_completion_with_tool_calling_streaming
+
+inference_providers:
+  - ollama
+  - fireworks
+  - together
+  - tgi
+  - vllm
```

Comment:
Are these inline or remote providers?

Reply:
Or is there a way to specify explicitly?

Reply:
@terrytangyuan I think these are references to the fixture names in the corresponding directories. given https://github.com/meta-llama/llama-stack/blob/main/llama_stack/providers/tests/inference/fixtures.py#L251, this should probably be `vllm_remote`

Reply:
Sounds good!

Reply:
@sixianyi0721 can you change this to `vllm_remote`?